src/store/chat/saga.ts

```diff
@@ -115,6 +122,14 @@ export function* performValidateActiveConversation(activeConversationId: string)
   if (!featureFlags.allowJoinRoom) {
     const isUserMemberOfActiveConversation = yield call(isMemberOfActiveConversation, activeConversationId);
     yield put(setIsConversationErrorDialogOpen(!isUserMemberOfActiveConversation));
+    if (!isUserMemberOfActiveConversation) {
+      yield put(
+        setJoinRoomErrorContent({
+          header: 'Access Denied',
+          body: 'You do not have permission to join this conversation.',
+        })
+      );
+    }
```

Comment:
it could also be the case that the room does not exist?

Reply:
> isMemberOfActiveConversation is false, this doesn't necessarily mean that the room does not exist.

Correct, the room can exist, but if the user isn't part of it, then we call the API where the user can request to join that room. But that's featureFlagged, and i see this error is thrown if the feature flag is not enabled.

I guess we can just update this error message to a more generic one - maybe `either the room doesn't exist, or you don't have permission to join it`? But fine for now, since this will be removed anyway.

Reply:
Generally speaking if you don't have permission then we should tell the user it doesn't exist. Much like if you try to view a github private repo but you're not logged in. This prevents people from just spamming urls until they find ids that exist.

Reply:
Also, given that the feature flag is only there while we wait for the api end to be completed, it's pretty meaningless what happens when it's disabled.